cmd/network.go

```diff
@@ -15,27 +16,59 @@ import (
 
 type networkCmd struct {
 	*config
-	*genericcli.GenericCLI[*models.V1NetworkCreateRequest, *models.V1NetworkUpdateRequest, *models.V1NetworkResponse]
 	childCLI *genericcli.GenericCLI[*models.V1NetworkAllocateRequest, any, *models.V1NetworkResponse]
 }
 
 func newNetworkCmd(c *config) *cobra.Command {
 	w := networkCmd{
-		config:     c,
-		GenericCLI: genericcli.NewGenericCLI[*models.V1NetworkCreateRequest, *models.V1NetworkUpdateRequest, *models.V1NetworkResponse](networkCRUD{config: c}),
-		childCLI:   genericcli.NewGenericCLI[*models.V1NetworkAllocateRequest, any, *models.V1NetworkResponse](networkChildCRUD{config: c}),
+		config:   c,
+		childCLI: genericcli.NewGenericCLI[*models.V1NetworkAllocateRequest, any, *models.V1NetworkResponse](networkChildCRUD{config: c}).WithFS(c.fs),
 	}
 
-	cmds := newDefaultCmds(&defaultCmdsConfig[*models.V1NetworkCreateRequest, *models.V1NetworkUpdateRequest, *models.V1NetworkResponse]{
-		gcli:     w.GenericCLI,
-		singular: "network",
-		plural:   "networks",
-
-		createRequestFromCLI: w.createRequestFromCLI,
-
-		availableSortKeys: sorters.NetworkSortKeys(),
-		validArgsFunc:     c.comp.NetworkListCompletion,
-	})
+	cmdsConfig := &genericcli.CmdsConfig[*models.V1NetworkCreateRequest, *models.V1NetworkUpdateRequest, *models.V1NetworkResponse]{
+		BinaryName:           binaryName,
+		GenericCLI:           genericcli.NewGenericCLI[*models.V1NetworkCreateRequest, *models.V1NetworkUpdateRequest, *models.V1NetworkResponse](w).WithFS(c.fs),
+		Singular:             "network",
+		Plural:               "networks",
+		Description:          "networks can be attached to a machine or firewall such that they can communicate with each other.",
+		CreateRequestFromCLI: w.createRequestFromCLI,
+		AvailableSortKeys:    sorters.NetworkSortKeys(),
+		ValidArgsFn:          c.comp.NetworkListCompletion,
+		DescribePrinter:      func() printers.Printer { return c.describePrinter },
+		ListPrinter:          func() printers.Printer { return c.listPrinter },
+		CreateCmdMutateFn: func(cmd *cobra.Command) {
+			cmd.Flags().StringP("id", "", "", "id of the network to create. [optional]")
+			cmd.Flags().StringP("description", "d", "", "description of the network to create. [optional]")
+			cmd.Flags().StringP("name", "n", "", "name of the network to create. [optional]")
+			cmd.Flags().StringP("partition", "p", "", "partition where this network should exist.")
+			cmd.Flags().StringP("project", "", "", "project of the network to create. [optional]")
+			cmd.Flags().StringSlice("prefixes", []string{}, "prefixes in this network.")
+			cmd.Flags().StringSlice("labels", []string{}, "add initial annotation, must be in the form of key=value, can be given multiple times to add multiple annotations, e.g. --annotation key=value --annotation foo=bar")
```

Comment:
changed from annotation to labels, but example still references to annotation, i think plural is wrong here, it seems weird to type `--labels key=value` should be `--label key=value`